sdk/daml-lf/transaction-test-lib/src/main/scala/com/digitalasset/daml/lf/value/test/ValueGenerators.scala

```diff
@@ -315,12 +316,16 @@ object ValueGenerators {
     for {
       coid <- coidGen
       packageName <- pkgNameGen
-      pkgVer <- pkgVerGen(version)
+      pkgVer <-
+        if (version <= LanguageVersion.v2_1) Gen.const(Option.empty[PackageVersion])
+        else pkgVerGen(version)
```

Comment:
@remyhaemmerle-da does it make sense to do this? I removed those fields from the proto serialization in canton, and I'd like to re-use those generators to test that the serialization code properly round trips transaction to / from proto. But it fails if the transactions contain those fields.